Makefile

```
@@ -134,6 +134,11 @@ deploy-community-policy-framework-managed: deploy-policy-framework-managed-crd-o
 kind-deploy-policy-framework:
 	@echo installing policy-propagator on hub
 	kubectl create ns $(KIND_HUB_NAMESPACE) --kubeconfig=$(PWD)/kubeconfig_$(HUB_CLUSTER_NAME)
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
```

Since this is the second place we're installing cert-manager, would you create a `CERT_MANAGER_VERSION` variable (or similarly named) to use both places so we only need to update the version in one place?
```suggestion
	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml
```
https://github.com/stolostron/governance-policy-framework/blob/a6e95b78550ca28fc38b5baa04b60458379d5f8a/Makefile#L157-L158

---

OH!! great idea